src/components/Footer.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+
+type FooterProps = {
+  todos: Todo[];
+  filterType: 'All' | 'Active' | 'Completed';
+  changeFilterStatus: (type: 'All' | 'Active' | 'Completed') => void;
+  handleClearCompleted: () => void;
+};
+
+export const Footer: React.FC<FooterProps> = ({
+  todos,
```

Comment:
Nitpick: This component does not need to receive the whole list as a prop. I always prefer to pass as little information as needed, so in this case I would calculate `remainingItems` and `completedItems` in parent element and pass down only the number. 
It is not required however, so you don't need to change it if you don't want to. 
I just wrote this for your consideretion.